project/dataset/valdo.py

```diff
@@ -55,4 +57,138 @@ def count_cmb_per_image(self, segmented_images):
             cmb_counts.append(total_cmb_count)
         return cmb_counts
     
+    # def get_cropped_locations(self, img, x_min, y_min, x_max, y_max):
```

Comment:
Most of these are commented out, I'll ignore these

Reply:
May mga changes si lex noon kasi derived yan sa original bounding  box function ng final project. May mga iba diya naka comment out na talaga.

Reply:
So pwede ko tanggalin?

Reply:
yap